civithermometer.php

```diff
@@ -163,31 +163,29 @@ function civithermometer_civicrm_tabset($tabsetName, &$tabs, $context) {
   if ($tabsetName == 'civicrm/admin/contribute') {
     if (!empty($context['contribution_page_id'])) {
       $contribID = $context['contribution_page_id'];
-      $url = CRM_Utils_System::url('civicrm/admin/contribute/thermometer',
-        "reset=1&snippet=5&force=1&id=$contribID&action=update&component=contribution");
       $tab['thermometer'] = array(
         'title' => ts('Thermometer'),
-        'link' => $url,
+        'link' => NULL,
         'valid' => 1,
         'active' => 1,
         'current' => false,
       );
+      // Insert this tab into position 2
+      $tabs = array_merge(
+        array_slice($tabs, 0, 2),
+        $tab,
+        array_slice($tabs, 2)
+       );
     }
     if (!empty($context['urlString']) && !empty($context['urlParams'])) {
-      $tab[] = array(
+      $tabs[CRM_Core_Action::VIEW] = array(
```

Comment:
the menu or as it is called in core Action Links the key is always a CRM_Core_Action::<constant>

Reply:
Fascinating. Not flagged anywhere in the documentation I came across at the time. I'll have to find it and propose an edit. Thanks @seamuslee001